measure/sbin/griffin-tool.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
As a suggestion, maybe we can have dedicated logging functions like below

```
# Logging functions

function log_info() {
  echo "[INFO] | $(date -u +"%D %T") UTC | $1"
}

function log_error() {
  echo "[ERROR] | $(date -u +"%D %T") UTC | $1"
}
```

and then we can replace all the `echo` with the above logging functions, for example,

```
if [ ! -f ${envFile} ];then
  envFile="${CONFDIR}/${envFile}"
  if [ ! -f ${envFile} ];then
    log_info "Not found env file: $1"
    exit
  fi
fi
```

This will help show the logs nicely.